src/modalities/config/config.py

```diff
@@ -203,18 +223,23 @@ class CosineAnnealingLRSchedulerConfig(BaseModel):
 
 
 class CheckpointedOptimizerConfig(BaseModel):
-    checkpoint_loading: PydanticCheckpointLoadingIFType
+    checkpoint_loading: PydanticFSDP1CheckpointLoadingIFType
     checkpoint_path: Path
     wrapped_model: PydanticPytorchModuleType
     optimizer: PydanticOptimizerIFType
 
 
-class CheckpointedModelConfig(BaseModel):
-    checkpoint_loading: PydanticCheckpointLoadingIFType
+class FSDP1CheckpointedModelConfig(BaseModel):
```

Comment:
Here is a maming inconsitency between CheckpointedOptimizerConfig and FSDP1CheckpointedModelConfig. Please name either both with FSDP1 prefix or none with the prefix.